math/caffe/files/patch-cudnn.diff

```diff
@@ -1,11 +1,11 @@
---- Makefile.config.orig	2015-02-22 14:13:41.000000000 +0900
-+++ Makefile.config	2015-02-22 14:15:21.000000000 +0900
+--- Makefile.config.orig	2017-08-24 10:43:43.000000000 +0900
++++ Makefile.config	2017-08-24 10:43:54.000000000 +0900
 @@ -2,7 +2,7 @@
  # Contributions simplifying and improving our build system are welcome!
- 
+
  # cuDNN acceleration switch (uncomment to build with cuDNN).
 -# USE_CUDNN := 1
-+USE_CUDNN := 1
- 
++ USE_CUDNN := 1
```

Comment:
Why an extra space?

Reply:
My fault, when I removed the commend hash by hand on the original source code, but missed the space between the hash and the variable.
I've summited the removal of that spaces.